backend/satellite_tools/reposync.py

```diff
@@ -813,6 +816,11 @@ def import_modules(self, plug):
         modulesfile = plug.get_modules()
         if modulesfile:
             self.copy_metadata_file(plug, modulesfile, 'modules', relative_modules_dir)
+    def import_mediaproducts(self, plug):
+        mediaproducts = plug.get_mediaproducts()
+        if mediaproducts:
+            self.copy_metadata_file(plug, mediaproducts, 'mediaproducts', relative_mediaproducts_dir)
+
```

Comment:
```suggestion

    def import_mediaproducts(self, plug):
        mediaproducts = plug.get_mediaproducts()
        if mediaproducts:
            self.copy_metadata_file(plug, mediaproducts, 'mediaproducts', relative_mediaproducts_dir)

```

Reply:
I do not see a difference? Is there a tab vs. space thingy?

Reply:
Got it. The extra line.